great-idea/index.html

```diff
@@ -38,7 +39,7 @@ <h1>Innovation<br> On<br> Demand</h1>
     </section>
```

Comment:
probably no need to have a "cta" and "cta-text". one container should be enough for this element.